projects/plugins/jetpack/modules/masterbar/profile-edit/profile-edit.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * WP-Admin Profile edit.
+ *
+ * @package Jetpack
+ */
+
+use Automattic\Jetpack\Connection\Manager as Connection_Manager;
+
+/**
+ * Hides profile fields for WordPress.com connected users.
+ *
+ * @param WP_User $user The current WP_User object.
+ */
+function masterbar_hide_profile_fields( $user ) {
```

Comment:
Nitpick: Given this file doesn't have a namespace, the function should be prefixed with `jetpack_` to avoid potential collisions with other plugins.

Reply:
Good catch 7eabddd!